image_width: 320
image_height: 240
camera_name: narrow_stereo
camera_matrix:
  rows: 3
  cols: 3
  data: [190.68123,   0.     , 160.50000,
           0.     , 190.68123, 120.50000,
           0.     ,   0.     ,   1.     ]
camera_model: plumb_bob
distortion_coefficients:
  rows: 1
  cols: 5
  data: [0.000000, 0.000000, 0.000000, 0.000000, 0.000000]
rectification_matrix:
  rows: 3
  cols: 3
  data: [1., 0., 0.,
         0., 1., 0.,
         0., 0., 1.]
projection_matrix:
  rows: 3
  cols: 4
  data: [190.68123,   0.     , 160.50000,   0.     ,
           0.     , 190.68123, 120.50000,   0.     ,
           0.     ,   0.     ,   1.     ,   0.     ]
